src/color/setting.js

```diff
@@ -177,44 +194,42 @@ p5.prototype.background = function(...args) {
 };
 
 /**
- * Clears the pixels within a buffer. This function only clears the canvas.
- * It will not clear objects created by createX() methods such as
- * <a href="#/p5/createVideo">createVideo()</a> or <a href="#/p5/createDiv">createDiv()</a>.
- * Unlike the main graphics context, pixels in additional graphics areas created
- * with <a href="#/p5/createGraphics">createGraphics()</a> can be entirely
- * or partially transparent. This function clears everything to make all of
- * the pixels 100% transparent.
- *
- * Note: In WebGL mode, this function can be passed normalized RGBA color values in
- * order to clear the screen to a specific color. In addition to color, it will also
- * clear the depth buffer. If you are not using the webGL renderer
+ * Clears the pixels within a buffer, specifically the canvas.
```

Comment:
Great, committed all these changes.

Reply:
Looks good!

I have a few more edits in mind. Would you like to work on them in this pull request or start working on your next contribution? If you're up for it, it'd be a huge help to start revising our [Examples](https://p5js.org/examples/) so that they follow the [style guide](https://github.com/processing/p5.js/blob/main/contributor_docs/documentation_style_guide.md).

Reply:
Yeah, let me know your edits and I can work on them in this pull request next week. I can also work on revising the examples.

Reply:
Let me know if you have any more suggested changes I can update over the next few days. And for the examples, is there already an open issue?

Reply:
I'll edit the next batch over the weekend. Please feel free to open an issue for revising Examples in the [website repo](https://github.com/processing/p5.js-website/).